learn/pkg/learn/export.go

```diff
@@ -195,11 +187,20 @@ func writeHTMLFile(model model, sidebarContent string, srcDir, destDir string, o
 	if err != nil {
 		return fmt.Errorf("%w: %w: %s", ErrInconsistentMdoel, err, model.Filename())
 	}
-	htmlFile := filepath.Join(destDir, md.HTMLFilename(mdFile))
-	content, err := model.ToHTML(opts.WithAnswersMarked)
+	content, err := model.ToHTML(opts.WithAnswersMarked) // without answers marked
```

Comment:
I dont understand this comment you've added - the parameter specifies whether to export with answers marked or not. How does this comment apply?

Reply:
it doesn't. removed, thank you.